hexrd/instrument.py

```diff
@@ -681,6 +681,10 @@ def __init__(self, instrument_config=None,
             for det_id, det_info in detectors_config.items():
                 pixel_info = det_info['pixels']
                 affine_info = det_info['transform']
+                if 'detector_type' in det_info:
+                    detector_type = det_info['detector_type']
+                else:
+                    detector_type = 'planar'
```

Comment:
Here, this is simpler:
```suggestion
                detector_type = det_info.get('detector_type', 'planar')
```